bedrock/mozorg/templates/mozorg/contentservices/dmexco.html

```diff
@@ -0,0 +1,63 @@
+{# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/. -#}
+
+{% extends "mozorg/contentservices/base.html" %}
+
+{% block page_title %}{{ _('@dmexco') }}{% endblock %}
+{% block body_id %}contentservices-dmexco{% endblock %}
+{% block body_class %}sand{% endblock %}
+
+{% block page_css %}
+  {% stylesheet 'contentservices' %}
+{% endblock %}
+
+{% block article %}
+  <h1 class="title-shadow-box">{{ self.page_title() }}</h1>
+
+  <div class="content-services-dmexco" itemscope itemtype="http://schema.org/Event">
+    <section>
+      <p itemprop="review">
+        <span itemprop="attendee">{{ _('Mozilla’s Content Services team') }}</span> {{ _('will be at <a href="http://dmexco.de/en/Home.html">dmexco 2015</a> to discuss our Tiles products and our vision for advancing user control on the Web.') }}
+        {{ _('We will be meeting with partners - both content partners and advertisers - who share our values and are looking for a future-facing media partner with global scale.') }}
+      </p>
+      <img itemprop="image" alt="dmexco logo" src="{{ static('img/mozorg/contentservices/dmexco_logo.jpg') }}">
+    </section>
+
+    <section>
+      <h2>{{ _('Location') }}</h2>
+      <p>{{ _('Come and find us at:')}}</p>
+      <a href="https://service.dmexco.de/km_vis-cgi/km_vis/vis/custom/ext2/show_exhibitor_data.cgi?ticket=k8366394685435&selected_by=Ausstellerkategorie%3A%2025%0A&exh_id=461" class="location" itemprop="location" itemscope itemtype="http://schema.org/Place">
+        <span>{{ _('Hall 7, Stand F-061') }}</span>
+        <span itemprop="name">{{ _('Dmexco 2015') }}</span>
+        <span>{{ _('Koelnmesse') }}</span>
+        <span>{{ _('Congress Centre North') }}</span>
+        <img itemprop="image" alt="dmexco map" src="{{ static('img/mozorg/contentservices/dmexco_map.png') }}">
+      </a>
+    </section>
+
+    <section>
+      <h2>{{ _('Booking a meeting') }}</h2>
+      <p itemscope itemtype="http://schema.org/ContactPoint">{{ _('You can book a meeting with us at the reception desk in our booth, or contact us in advance:') }} <a itemprop="email" href="emailto:contentservices@mozilla.com">contentservices@mozilla.com</a></p>
+    </section>
+
+    <section>
+      <h2>{{ _('Speaking') }}</h2>
+      <div class="sub-event">
+        <p>
+          <a href="https://blog.mozilla.org/press/bios/darren-herman/" itemprop="performer">Darren Herman</a>, {{ _('VP Content Services, will be speaking on the topic of ')}}
+          <em>{{ _('The future of native advertising and content marketing.') }}</em>
+        </p>
+        <strong itemprop="subEvent">{{ _('Marketing Track, Seminar 3') }}</strong>
+        <p itemprop="startDate">{{ _('September 16, 13:00-13:45') }}</p>
```

Comment:
Could use `<time>` here instead of a `<p>`? 

If you want to go the extra mile you could also add a valid `datetime` attribute, to make it more machine-readable? :smile: